packages/pluggableWidgets/tree-view-web/package.json

```diff
@@ -32,7 +32,12 @@
   },
   "devDependencies": {
     "@mendix/pluggable-widgets-tools": ">=9.0.0",
+    "@testing-library/dom": "^8.1.0",
```

Comment:
I think we should move theme dependencies to PIW-tools or also include in the root package.

Reply:
I will move them to PIW-tools

Reply:
fixed